deployment/ansible/roles/k8s/tasks/components/kuma.yml

```diff
@@ -0,0 +1,102 @@
+---
+- name: Install argocd
+  kubernetes.core.helm:
+    chart_repo_url: https://dirsigler.github.io/uptime-kuma-helm
+    chart_ref: uptime-kuma
+    chart_version: 2.5.4
+    release_name: uptime-kuma
+    namespace: monitoring
+    create_namespace: yes
+- name: Make shure kuma has an ingress
+  kubernetes.core.k8s:
+    state: present
+    namespace: monitoring
+    definition:
+      apiVersion: traefik.containo.us/v1alpha1
+      kind: IngressRoute
+      metadata:
+        name: kuma-uptime
+        namespace: monitoring
+      spec:
+        entryPoints:
+          - websecure
+        routes:
+          # temporary workaround for traefik bug (interoperability with ingress)
+          - kind: Rule
+            match: Host(`uptime.nav.tum.sexy`) && PathPrefix(`/.well-known/acme-challenge/`)
+            priority: 13
+            services:
+              - name: cm-acme-http-solver-mppjd
+                port: 8089
+                kind: Service
+          - kind: Rule
+            match: Host(`uptime.nav.tum.sexy`)
+            priority: 12
+            services:
+              - name: uptime-kuma
+                port: 3001
+        tls:
+          secretName: uptime.nav.tum.sexy
+      
+- name: Make shure kuma has an http-ingress
+  kubernetes.core.k8s:
+    state: present
+    namespace: monitoring
+    definition:
+      apiVersion: traefik.containo.us/v1alpha1
+      kind: IngressRoute
+      metadata:
+        name: kuma-uptime-http
+        namespace: monitoring
+      spec:
+        entryPoints:
+          - web
+        routes:
+          # temporary workaround for traefik bug (interoperability with ingress)
+          - kind: Rule
+            match: Host(`uptime.nav.tum.sexy`) && PathPrefix(`/.well-known/acme-challenge/`)
+            services:
+              - name: cm-acme-http-solver-mppjd
+                port: 8089
+                kind: Service
+          - kind: Rule
+            match: Host(`uptime.nav.tum.sexy`)
+            middlewares:
+              - name: https
+            services:
+              - kind: TraefikService
+                name: noop@internal
+      
+- name: Make shure kuma has an http redirect
+  kubernetes.core.k8s:
+    state: present
+    namespace: monitoring
+    definition:
+      apiVersion: traefik.containo.us/v1alpha1
+      kind: Middleware
+      metadata:
+        name: https
+        namespace: monitoring
+      spec:
+        redirectScheme:
+          scheme: https
+          permanent: true
+
+- name: Make shure kuma has an cert
+  kubernetes.core.k8s:
+    state: present
+    namespace: monitoring
+    definition:
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        name: uptime.nav.tum.sexy
+        namespace: monitoring
+      spec:
+        commonName: uptime.nav.tum.sexy
+        dnsNames:
+          - uptime.nav.tum.sexy
+        secretName: uptime.nav.tum.sexy
```

Comment:
If we still have it: Probably nav.tum.de now

Reply:
see above